uber/config.py

```diff
@@ -157,6 +157,42 @@ def BADGES_SOLD(self):
             return individuals + group_badges
 
     @property
+    def BADGES_LEFT_AT_CURRENT_PRICE(self):
+        """
+        Returns a string representing a rough estimate of how many badges are left at the current badge price tier.
+        """
+        if c.HARDCORE_OPTIMIZATIONS_ENABLED:
+            return 'Unknown'
+
+        current_price_tier = c.ORDERED_PRICE_LIMITS.index(c.BADGE_PRICE) if c.BADGE_PRICE in c.ORDERED_PRICE_LIMITS else -1
+        if current_price_tier != -1 and c.ORDERED_PRICE_LIMITS[current_price_tier] == c.ORDERED_PRICE_LIMITS[-1]\
+                or not c.ORDERED_PRICE_LIMITS:
+            if c.MAX_BADGE_SALES:
+                difference = c.MAX_BADGE_SALES - c.BADGES_SOLD
+            else:
+                return 'Limitless'
+        else:
+            for key, val in c.PRICE_LIMITS.items():
+                if c.ORDERED_PRICE_LIMITS[current_price_tier+1] == val:
+                    difference = key - c.BADGES_SOLD
+        if not difference:
+            return 'Unknown'
+        elif difference <= 50:
+            return 'Almost Gone'
+        elif difference <= 100:
+            return 'Very Low'
+        elif difference <= 250:
+            return 'Low'
+        elif difference <= 500:
+            return 'Medium'
+        elif difference <= 750:
+            return 'High'
+        elif difference <= 1000:
+            return 'Very High'
+        else:
+            return 'Super High'
```

Comment:
Just make it return an numeric value and have a second function that determines the label. If a plugin wants to change the labels, it can override that function.